src/System.Private.CoreLib/src/System/TypeLoadException.CoreCLR.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+**
+**
+** Purpose: The exception class for type loading failures.
+**
+**
+=============================================================================*/
+
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
This should be in the share part.

Reply:
Both Mono and CoreRT don't have it, should I still move the attribute there?

Reply:
CoreRT has it: https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System/TypeLoadException.cs#L10

And all exceptions in shared have it too.

Reply:
ah indeed